test/java/org/opendatakit/briefcase/model/ServerConnectionInfoTest.java

```diff
@@ -0,0 +1,34 @@
+package org.opendatakit.briefcase.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ServerConnectionInfoTest {
```

Comment:
This tests are only increasing the coverage percentage, which is not useful by itself. This test class doesn't bring much value to the table because equality rules of ServerConnectionInfo are indirectly verified by higher-level of abstraction tests elsewhere. I'd suggest removing this test class